params/robot_localization_config.yaml

```diff
@@ -150,6 +156,15 @@ twist0_config: [false, false, false,
 twist0_queue_size: 3
 twist0_nodelay: false
 
+twist1: optical_flow_estimator/twist
```

Comment:
I noticed that none of our z estimates are being fused as a velocity. I believe we were doing that for a while, why did we stop?

Reply:
IIRC we were fusing z estimates as velocities before we had accelerations from the imu because it was the only way to get a reasonable velocity, and we stopped once we added the imu